templates/zerver/help/saml-authentication.md

```diff
@@ -49,7 +49,10 @@ providers.
 1. {!send-us-info.md!}
 
      1. Your organization's URL
-     1. The **Identity Provider metadata** provided by Okta for the application
+     1. The **Identity Provider metadata** provided by Okta for the application.
+        You can find it by going to the **Sign on** tab, and using the
+        **View SAML setup instructions** button found in the right sidebar.
+        The metadata is at the bottom of the new page that opens.
```

Comment:
How about  replacing 

> You can find it by going to the **Sign on** tab, and using the
        **View SAML setup instructions** button found in the right sidebar.

with 

> The **Identity Provider metadata** provided by Okta for the application. To get the data, click the
        **View SAML setup instructions** button in the right sidebar on the**Sign on** tab.

?

Reply:
Yeah, that sounds nicer, Amended the commit to be like this:
```
     1. The **Identity Provider metadata** provided by Okta for the application.
        To get the data, click the **View SAML setup instructions button** in
        the right sidebar in the **Sign on** tab.
        The metadata is at the bottom of the new page that opens.
```

Reply:
How about this?

> The metadata is at the bottom of the new page that opens.

->

> Copy the **IDP metadata** shown at the bottom of the page.

Reply:
Sure, sounds good!